### ISSUE TITLE
Remove the "Ecommerce" prefixing

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/traceability-vocab#113]
The vocab is for supply chain... we don't need "Ecommerce" prefixed everywhere.. and it makes things less reusable.

**Comment by Therecanbeonlyone1969** [comment]
@OR13 ... happy to do that if we can standardize on the things that are must requirements for e-commerce which is currently represented in the VCs and evidence documents.

**Comment by mprorock** [comment]
Agreed.  Good topic for discussion.   I think there is a place for
prefixing, but we should standardize when and where, and add to readme

**Comment by bumblefudge** [comment]
Does "whole cohort" mean supply chain traceability subcohort or SVIP-wide cohort? 

I think it might also be a topic for a CCG community call or DIF Interop WG, as I would be curious to poll implementers if there are best practices or conventions that might help?

**Comment by OR13** [comment]
@Therecanbeonlyone1969 any chance you can tackle this cleanup?

**Comment by Therecanbeonlyone1969** [comment]
@OR13 yup ... will do a PR. What do we do for Orders and Invoices? We have conflicting schemas there. Wanna get on a call with interesting ppl and hash it out? cc @mprorock @bumblefudge

**Comment by OR13** [comment]
If we remove prefixes, then we need a base template, for shareable types and extensions.... blocked by base types?

**Comment by OR13** [comment]
blocked by https://github.com/w3c-ccg/traceability-vocab/issues/162

**Comment by OR13** [comment]
Proposal to add base types and removing prefixes in a single PR.

**Comment by OR13** [comment]
I think we can safely remove the prefix, without discussing other issues.

**Comment by OR13** [comment]
If there exist no conflict, remove the prefix.

if there exists a conflict, raise a separate issue.

**Comment by mprorock** [comment]
Per call: Can be revisted post YAML conversion now

**Comment by OR13** [comment]
we also have an issue with the evidence property. I suggest we just do our best to. simplify as much as we can.

**Comment by OR13** [comment]
@BenjaminMoe can you take a stab at this.

**Comment by BenjaminMoe** [comment]
Yep! On it!
